### PR TITLE
chore(github) Added github issue and pull request templates.

### PR DESCRIPTION
**Pull request by TheLarkInn**
This PR adds github issue templating and pull request templating.

### CODE REVIEW

**Comment by TheLarkInn** [comment]
@sokra @bebraw @jhnns Let me know if you want to style this differently. I tried to take a generic approach on this one.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#### BEFORE YOU SUBIMIT please read the following:
```
Comment (bebraw) [inline .github/ISSUE_TEMPLATE.md]: /s/SUBIMIT/SUBMIT

Maybe `##` over `####`. Not sure.

Comment (bebraw) [inline .github/ISSUE_TEMPLATE.md]: You could also do bolding instead. That would be noticeable too.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] bug report
+- [ ] feature request
+- [ ] support request => Please do not submit support request here, see note at the top of this template. 
+
+
+** Webpack version: **
+1.10.x/2.x
+
+** Please tell us about your environment: **
+OSX 10.x / Linux / Windows 10
```
Comment (bebraw) [inline .github/ISSUE_TEMPLATE.md]: Checkbox here too?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (TheLarkInn) [inline .github/ISSUE_TEMPLATE.md]: Im fine with making it larger.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (TheLarkInn) [inline .github/ISSUE_TEMPLATE.md]: I think this lets people "check" the box themself. I can't remember the markdown for the checkbox.

**Comment by TheLarkInn** [comment]
https://github.com/TheLarkInn/webpack/blob/fe7cc810f5d054e3f784f0c6cb5fb2b574ce78fd/.github/ISSUE_TEMPLATE.md Alright I fixed those checkboxes. Was missing a space. @bebraw

**Comment by TheLarkInn** [comment]
I can clean that up in the feature request as well.

**Comment by TheLarkInn** [comment]
I can clean that up in the feature request as well.

**Comment by jhnns** [comment]
LGTM @TheLarkInn 

Could you also add to the `PULL_REQUEST_TEMPLATE.md` that bigger features and especial breaking changes should have a corresponding issue with a discussion.

I hope these templates won't scare real bug reporters :)

**Comment by TheLarkInn** [comment]
Yup I can get that in soon.

**Comment by TheLarkInn** [comment]
Anything else you can think of?

**Comment by bebraw** [comment]
You could fix that `SUBIMIT` typo.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+**Does this PR introduce a breaking change?**
+- [ ] Yes
+- [ ] No
+
+If this PR contains a breaking change, please describe the following... 
+
+* Impact:
+* Migration path for existing applications: 
+* Gihub Issue(s) this is regarding:
```
Comment (also) [inline .github/PULL_REQUEST_TEMPLATE.md]: There's a typo in "Gihub"

Comment (TheLarkInn) [inline .github/PULL_REQUEST_TEMPLATE.md]: Thanks I'll correct that.

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+I'm submitting a...
+=====
+- [ ] bug report
+- [ ] feature request
+- [ ] support request => Please do not submit support request here, see note at the top of this template. 
+
+
+** Webpack version: **
```
Comment (also) [inline .github/ISSUE_TEMPLATE.md]: `** Webpack version: **` doesn't render right because of the spaces inside the asterisks:

** Webpack version: **

**Comment by jhnns** [comment]
thx @TheLarkInn

**Pull request merged**